src/resources/Sources/SourcesInterfaces.ts

```diff
@@ -28,6 +29,10 @@ export interface ListOperationalStatusSourcesParams extends ListParams {
     sourceOperationalStatus: SourceOperationalStatus;
 }
 
+export interface GetDefaultDocumentConfigurationParams {
```

Comment:
```suggestion
export interface DefaultDocumentConfigurationParams {
```
IMO, the interface should say what it is, not what it does